code/modules/mob/living/brain/posibrain.dm

```diff
@@ -256,17 +256,16 @@ GLOBAL_VAR(posibrain_notify_cooldown)
 	if(ismovable(loc) && prob(25))
 		var/obj/item/item = pick(loc.contents)
 		if(istype(loc, /obj/item/storage))
-			item.forceMove(get_turf(src)) //throw stuff out of the inventory till we free ourselves!
+			item.forceMove(loc.drop_location()) //throw stuff out of the inventory till we free ourselves!
 			playsound(src, SFX_RUSTLE, 30, TRUE)
+			return
```

Comment:
```suggestion
		return
```
We can return even if you're not in a storage, we've figured out that we're also not a turf.
Still means that 75% of the time that we're in a movable then we will check if we're in a turf anyway but... eh who cares